packages/jest-diff/src/types.ts

```diff
@@ -41,4 +42,5 @@ export type DiffOptionsNormalized = {
   omitAnnotationLines: boolean;
   patchColor: DiffOptionsColor;
   trailingSpaceFormatter: DiffOptionsColor;
+  trimmableLineReplacement: string;
```

Comment:
What do you think about something like `surroundingEmptyLineReplacement`?
I couldn't immediately think of what `trimmableLineReplacement` might mean.

Reply:
Constructive suggestion as usual. After some unsuccessful search for additional synonyms, I found that `firstOrLastEmptyLineReplacement` has same length. Also factored out that logic, because the original code assumed the default value of `commonIndicator` option.